tests/Intl/Normalizer/NormalizerTest.php

```diff
@@ -44,14 +44,14 @@ public function testIsNormalized()
         $c = 'déjà';
         $d = in::normalize($c, pn::NFD);
 
-        $this->assertTrue(pn::isNormalized(''));
-        $this->assertTrue(pn::isNormalized('abc'));
-        $this->assertTrue(pn::isNormalized($c));
-        $this->assertTrue(pn::isNormalized($c, pn::NFC));
+        $this->assertTrue(!!normalizer_is_normalized(''));
```

Comment:
I suggest adding a comment referencing the HHVM issue, to remember why this casting is there.

And I think ``(bool)`` would be more readable for PHP devs (the ``!!`` is common in JS to save bytes, but you are the only PHP dev I know who try to make the PHP code look like minified code)